### ISSUE TITLE
Is there documentation or are there jupyter notebooks for Bandit?

### ISSUE THREAD

**Issue by dpphat** [paknorton/Bandit#22]
I would really like to use this application to access NHM datasets.
Thanks so much.

**Comment by pnorton-usgs** [comment]
There is no formal documentation yet. I've attached a pdf of instructions for using bandit with the National Hydrologic Model v1.0 that I've provided to others in the past along with a zipfile containing a conda yaml for the bandit python environment, a default control file, and a starting bandit config file. Depending on your needs the greatest challenge is the source data requirements (much of which is behind a firewall) for running a complete Bandit extraction including climate-by-HRU (CBH) files, dynamic parameters, model output, and GIS shapefiles. Below are some estimates of storage requirements for some of the source data. I hope to change this in the future have bandit pull source data from S3 buckets.

NHMparamdb (https://github.com/nhm-params-v10-usgs): 500-600 M
CBH files: Daymet (1980–2016) - 8.3 G; Maurer (1949–2010) - 16 G; <and others>
GIS files: 817 M (compressed)
NHM model output files: about 161 G per version (e.g. uncalibrated, calibrated byHRU, etc)
Dynamic parameters: 220 M (cov_type, hru_percent_imperv, snow_intcp, srain_intcp, and wrain_intcp)

For minimal working extractions you would need the NHMparamdb and a set of CBH files. Streamflow observations are pulled directly from the National Water Information System (NWIS; https://waterdata.usgs.gov). I can work with you to get the source datasets you nee
[01_Bandit setup.pdf](https://github.com/paknorton/Bandit/files/7588462/01_Bandit.setup.pdf)
[bandit_setup_file.zip](https://github.com/paknorton/Bandit/files/7588479/bandit_setup_file.zip)
d.
